test/conformance/device_code/linker_error.cpp

```diff
@@ -0,0 +1,30 @@
+// Copyright (C) 2024 Intel Corporation
+// Part of the Unified-Runtime Project, under the Apache License v2.0 with LLVM Exceptions.
+// See LICENSE.TXT
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+#include <CL/sycl.hpp>
+
+SYCL_EXTERNAL void this_function_does_not_exist();
+
+int main() {
+    cl::sycl::queue deviceQueue;
```

Comment:
```suggestion
    sycl::queue deviceQueue;
```

We should probably be using SYCL 2020 syntax for good measure.